### ISSUE TITLE
Style guide | Error behavior in forms

### ISSUE THREAD

**Issue by carodew** [department-of-veterans-affairs/caseflow#13838]
@gnakm commented on [Wed Nov 15 2017](https://github.com/department-of-veterans-affairs/appeals-team/issues/563)

## Description
Today across our caseflow apps, we use a disabled button when all required fields are not completed. 
- It has not always tested well. There has been instances where we've seen users click the disabled button
- It is not 508 compliant. When using a screen reader, the disabled button will get missed and goes in a loop where a user cannot identify the actual error
- It is redundant to have it with a form field error

## AC
- [ ] Investigate where we are currently using disabled buttons across products (Dispatch it one)
- [ ] Verify that applications with disabled buttons now show buttons in active state
- [ ] When required fields are missed, verify that there is a red error alert at the top listing the required fields that were missed
- [ ] When required fields are missed, verify that the form fields are displayed in their error states
- [ ] Get rid of all required messages and only put message for optional
- [ ] Remove [Disabled button in style guide ](http://dsva-appeals-certification-demo-1715715888.us-gov-west-1.elb.amazonaws.com/styleguide#disabled)
